flang/test/Lower/OpenMP/map-types-and-sizes.f90

```diff
@@ -0,0 +1,52 @@
+!RUN: %flang_fc1 -emit-llvm -fopenmp %s -o - | FileCheck %s
```

Comment:
LLVM tests should not be here. We only have lowering (parse-tree to MLIR) here.

Reply:
Can you please suggest a location for this test?

Is `flang/test/Driver/OpenMP/map-types-and-sizes.f90` suitable?

Reply:
OK.